ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -72,6 +73,19 @@ def BitwiseOr(*, x, y, name="BitwiseOr"):
     return ivy.bitwise_or(x, y)
 
 
+@to_ivy_arrays_and_back
+def avg_pool_3d(x, ksize, strides, padding, name=None):
+    with tf.name_scope(name or 'avg_pool_3d'):
```

Comment:
You have imported tensorflow and using `tf.name_scope()` here, the frontend is in tensorflow here, the implementation should be in `ivy`, which means you can only use ivy and no other framework :slightly_smiling_face:

Reply:
Okey, I removed that and found it's still working without it because I assigned  name above